terraform/aws/modules/customer-web-server/web-server.tf

```diff
@@ -424,6 +415,10 @@ data "kubernetes_service" "nginx-public" {
     name      = "mattermost-cm-nginx-public-nginx-ingress-controller"
     namespace = "network-public"
   }
+  # # Depends_on should be enabled on the first run of CWS deployment
```

Comment:
remove the comment if we dont need

Reply:
I left the comment just to know in the future when we want to enable the `depends_on`.
If we leave it there, in every plan it will have to check it if it is done